configen/OptionsParser.swift

```diff
@@ -54,24 +55,21 @@ final class OptionsParser {
 
     return plistDictionary
     }()
-
-  lazy var hintsDictionary: [String: String] = { [unowned self] in
+  
+  lazy var sortedHintsTuple: [Hint] = { [unowned self] in
```

Comment:
Thoughts on renaming this to `sortedHints`? Just because this is an array of tuples, not a tuple

Reply:
Ah yeah. Have updated. Thanks